assistant/gpt_assistant.py

```diff
@@ -373,6 +373,7 @@ def __init__(self, client: openai.AsyncOpenAI):
     async def send_to_assistant(
         self,
         prompt: str,
+        noa_system_prompt: str|None,
```

Comment:
Please add whitespace to match the formatting of the other params.

Reply:
done https://github.com/brilliantlabsAR/noa-assistant/pull/9/commits/86b7b9bdb3f875301f2412a1b52be7446164c1a4